Jellyfin.Drawing.Skia/SkiaCodecException.cs

```diff
@@ -6,18 +7,13 @@ namespace Jellyfin.Drawing.Skia
     /// <summary>
     /// Represents errors that occur during interaction with Skia codecs.
     /// </summary>
+    [SuppressMessage("Design", "CA1032:Implement standard exception constructors", Justification = "A custom property, CodecResult, is required when creating this exception type.")]
```

Comment:
@Bond-009 Is suppressing warnings via inline attributes our preferred approach when applicable? Or do we prefer `GlobalSuppressions.cs` or `#pragma warning disable`?

Reply:
This one should be added in the ruleset file imo (`jellyfin.ruleset`) as `Info`, you often don't want all ctors and suppressing each one individually isn't the cleanest thing ever.

Reply:
Addressed in 47805d8